test/CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ set(CORE_TEST_FILES
 foreach(i ${CORE_TEST_FILES})
   add_executable(${i} ${i}.cc)
   add_test(${i} ${i})
-  target_link_libraries(${i} ${GOOGLE_LIBS} tc_core tc_lang)
+  target_link_libraries(${i} ${GOOGLE_LIBS} tc_core tc_core_cpu tc_lang)
```

Comment:
do we need the cpu backend lib it for _all_ core tests?

Reply:
well it's 3 of the core tests, we can split in 2+1 but is it worth it at this point?